vendor/github.com/go-sql-driver/mysql/go.mod

```diff
@@ -1,3 +0,0 @@
-module github.com/go-sql-driver/mysql
```

Comment:
@timvaillancourt   what am I looking at here?

Reply:
@FromAshWeRise the files in `vendor/` are the result of `go mod vendor` + `git add vendor --all`, so `go mod` wanted this somehow